packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec

```diff
@@ -2,7 +2,7 @@
 %global gem_name fog-vsphere
 
 Name: rubygem-%{gem_name}
-Version: 3.5.2
+Version: 3.6.0
```

Comment:
Looking at https://github.com/theforeman/foreman-packaging/pull/8856 you're missing some updates to the minimum Ruby version. Are you using `./bump_rpm.sh` to do updates?